main.js

```diff
@@ -0,0 +1,175 @@
+function start() {
+  document.getElementById('cell-0').addEventListener('click', () => {
+    placeMark(0);
+    winChecker();
+  },
+    {once : true}
+  );
+  document.getElementById('cell-1').addEventListener('click', () => {
+    placeMark(1);
+    winChecker();
+  },
+  {once : true}
+  ); 
+  document.getElementById('cell-2').addEventListener('click', () => {
+    placeMark(2);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-3').addEventListener('click', () => {
+    placeMark(3);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-4').addEventListener('click', () => {
+    placeMark(4);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-5').addEventListener('click', () => {
+    placeMark(5);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-6').addEventListener('click', () => {
+    placeMark(6);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-7').addEventListener('click', () => {
+    placeMark(7);
+    winChecker();
+  },
+  {once : true}
+  );
+  document.getElementById('cell-8').addEventListener('click', () => {
+    placeMark(8);
+    winChecker();
+  },
+  {once : true}
+  );
+}
+// | - - - win check functions - - - |
+function winChecker () {
+  for (let subCombo of winCombo) {
+    let xCount = 0; // count of 3 signifies win 
+    let oCount = 0;
+    for (let num of subCombo) {
+      if (xLog.includes(num)) {
+        xCount = xCount + 1;
+      } else if (oLog.includes(num)){
+        oCount = oCount + 1; 
+      }
+    }
+    if (xCount == 3) {
+      gameStatus.textContent = playerNameX + " wins!";
+      showWin(xLog);
+      setTimeout(() => {location.reload(); }, 1000)
+    } else if (oCount == 3) {
+      gameStatus.textContent = playerNameO+ " wins!";
+      setTimeout(() => {location.reload(); }, 1000);
+      showWin(oLog);
+    } else if (moveCount.length === 9) {
+      gameStatus.textContent = "It's a draw";
+      setTimeout(() => {location.reload(); }, 1000);
+    }
+  }
+}
+function showWin (winningArray) {
+  for (let i = 0; i < winningArray.length; i++) {
+    document.getElementById('cell-' + winningArray[i]).style.background = "red"
+  }
+}
+// | - - - place mark on board & change player - - - |
+function placeMark(numCell) {
+  if (playerO) {
+    document.getElementById('cell-' + numCell).textContent = "0";
+    oLog.push(numCell);
+    oLog.sort();
+    moveCount.push(numCell);
+    playerO = false; // switch players
+    gameStatus.textContent = playerNameX + "'s turn";
+  }
+  else if (playerO === false) {
+    document.getElementById('cell-' + numCell).textContent = "X";
+    xLog.push(numCell);
+    xLog.sort();
+    moveCount.push(numCell);
+    playerO = true; // switch players
+    gameStatus.textContent = playerNameO + "'s turn";
+  }
+}
+// | - - - computer player functionality - - - |
+function activateComputer() {
```

Comment:
When creating an AI opponent, think all the way back to our "Guess the Number" project. You're going to want it to select a **random** unused square, (or, if you're willing to go the extra mile, the "best" possible square, which might be handled in a similar way that you handled your "winCombo" array) printing an **X** if the user is O, or **else** an O if the situation is reversed.